src/shared/api/urls.ts

```diff
@@ -208,7 +208,7 @@ export function getProxyUrlIfNecessary(url: any) {
 }
 
 export function getOncoKbApiUrl() {
-    return buildCBioPortalAPIUrl(`proxy/oncokb`);
+    return buildCBioPortalAPIUrl(`proxy/nothingToSeeHere`);
```

Comment:
@onursumer i think the idea here was that we would also mask the path.   but if that's problematic we could change this to something which doesn't attract attention.   this shows up in network tab so, kind of defeats the purpose
![image](https://user-images.githubusercontent.com/186521/156628369-e25acb8a-410d-4c1c-b926-321fea52edd8.png)

Reply:
We can do `proxy/x3345QR` but we need a new backend PR for that. Do you think we should also get rid of `proxy`?

Reply:
sure, if it's all the same, why not get rid of that too.

Reply:
and yes, we should do new backend PR

Reply:
@alisman Looks like it is not straightforward to get rid of proxy. We need to create a new endpoint class, and also move shared private methods to a utility class or something like that. Let's keep proxy for now?